client/my-sites/stats/stats-countries/style.scss

```diff
@@ -6,13 +6,17 @@ $break-large-stats-countries: 1280px;
 .stats__module-list--traffic {
 	.stats__module-wrapper--countryviews .stats-module,
 	.list-countryviews {
-		display: grid;
 		gap: 0 24px;
 		grid-template-columns: 7fr 5fr;
 		grid-template-areas:
 			"map list"
 			"more more";
 
+		// only applies this statement if there's data to show
+		&:has(.stats-card--hero .stats-geochart:not(.has-no-data)) {
```

Comment:
We should avoid using the CSS pseudo-class `has` since it is [not compatible with Firefox](https://developer.mozilla.org/en-US/docs/Web/CSS/:has#browser_compatibility).

Reply:
You're right. Thanks for pointing out this.

Reply:
@dognose24 just fixed this. I would appreciate if you could take another look at it.